prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -393,9 +393,9 @@ func Run(o *Options) error {
 		// If failed to push because a closed PR already exists with this change ID (the PR was abandoned). Hash the ID again and try one more time.
 		if err != nil && strings.Contains(err.Error(), "failed to push some refs") && strings.Contains(err.Error(), "closed") {
 			logrus.Warn("Error pushing CR due to already used ChangeID. PR may have been abandoned. Trying again with new ChangeID.")
-			changeId, err := getChangeId(o.Gerrit.Author, o.Gerrit.AutobumpPRIdentifier, changeId)
-			if err != nil {
-				return err
+			changeId, subErr := getChangeId(o.Gerrit.Author, o.Gerrit.AutobumpPRIdentifier, changeId)
```

Comment:
A couple of lines below is a statement that wants to set the `err` variable outside of the conditions scope, which didn't work because we shadowed the variable in the conditions scope

Reply:
@mpherman2 for awareness